lib/core/src/lib/comments/comment-list/comment-list.component.spec.ts

```diff
@@ -163,4 +163,86 @@ describe('CommentListComponent', () => {
         const elements = testingUtils.getAllByCSS('.adf-comment-user-icon');
         expect(elements.length).toBe(1);
     });
+
+    it('should display image using pictureId if available', async () => {
+        const comment = new CommentModel({
+            id: 1,
+            message: 'With pictureId only',
+            created: new Date(),
+            createdBy: {
+                firstName: 'Pic',
+                lastName: 'User',
+                pictureId: 1001
+            }
+        });
+
+        commentList.comments = [comment];
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        const img = testingUtils.getByCSS('.adf-people-img');
```

Comment:
We are looking for that selector multiple times in that file so could you please wrap ```testingUtils.getByCSS('.adf-people-img')``` in separate function to avoid repeating selector?